packages/extension/src/language/main.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { startLanguageServer } from 'langium/lsp';
+import { NodeFileSystem } from 'langium/node';
```

Comment:
Will this run in a [browser based lsp](https://github.com/microsoft/vscode-extension-samples/blob/main/lsp-web-extension-sample)?

Reply:
Nope, definitely not as is. We have an `EmptyFileSystem` that use for that purpose instead (it's a drop-in replacement), however this is the default that comes with a generated langium project, and it will be changed when we get to it. Same as the other points, a lot of this stuff here isn't really suitable for review yet, which is why I closed this out as soon as I noticed it wasn't on the fork. However, these changes (and those that will follow) will be coming over later in July/ early August, to align with our planned first spike.